java-to-zod-gradle-plugin/build.gradle.kts

```diff
@@ -0,0 +1,60 @@
+plugins {
```

Comment:
Although this is a Gradle plugin, I would like to see this built with Maven to keep consistency in the project. It looks like typescript-generator has accomplished this. I think this is mostly encapsulated in that sub-module's pom.xml: https://github.com/vojtechhabarta/typescript-generator/blob/main/typescript-generator-gradle-plugin/pom.xml
As well as this META-INF file: https://github.com/vojtechhabarta/typescript-generator/blob/main/typescript-generator-gradle-plugin/src/main/resources/META-INF/gradle-plugins/cz.habarta.typescript-generator.properties